.gitignore

```diff
@@ -123,6 +123,19 @@ fabric.properties
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
+### JDK ###
+.idea
+out
+server
+*.vsix
+.DS_Store
+undefined
```

Comment:
See above

Reply:
This gitignore file worked for me previously. If it has any unintended effects we can revise it later but I don't think there should be any issues.I'd like to avoid having random computer generated files on git. That is just prone to causing errors. For example, in the portfolio project, I ran into an problem where my code was compiling with errors because of an old target directory. Not sending these files to other peoples systems will probably be the best course of action. Also, for context, this is an official JDK gitignore file.